reference-app/src/main/java/org/smartregister/sample/fp/login/ui/LoginActivity.java

```diff
@@ -45,6 +42,7 @@ public void goToHome(boolean remote) {
     }
 
     private void gotToHomeRegister(boolean remote) {
+        ArchivedPostSterilizationJob.makeSchedule();
```

Comment:
Just wondering out loud, is there a reason why this was not placed in `org.smartregister.sample.fp.login.interactor.LoginInteractor#scheduleJobsPeriodically` like the scheduling for other jobs?

Reply:
Thanks, I haven't seen that method that's why I have misplaced that schedule but I move that.